src/components/NotificationError/NotificationError.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ErrorAction } from '../../types/ErrorAction';
+
+type Props = {
+  action: ErrorAction;
+  resetError: () => void;
+};
+
+export const NotificationError: React.FC<Props> = ({ action, resetError }) => {
+  const [isNotificationVisible, setIsNotificationVisible] = useState(true);
```

Comment:
you don't need this state variable because the component won't be rendered if there is no error (you have the check in `App.tsx`)
```suggestion
```